src/main/java/io/kontur/disasterninja/client/LayersApiClient.java

```diff
@@ -302,12 +301,12 @@ private Layer convertToLayerDetails(Geometry geoJSON, Collection collection, UUI
         if (collection == null) {
             return null;
         }
-        LayerSource source = null;
-        if (LAYER_TYPE_TILES.equals(collection.getItemType())) {
-            source = createVectorSource(collection);
-        } else if (LAYER_TYPE_FEATURE.equals(collection.getItemType())) {
-            source = createFeatureSource(geoJSON, collection.getId(), appId);
-        }
+        LayerSource source = switch (collection.getItemType()) {
+            case (LAYER_TYPE_TILES) -> createVectorSource(collection);
+            case (LAYER_TYPE_RASTER) -> createRasterSource(collection);
```

Comment:
Vector type was introduced in layers api as well. `tiles` type will become `vector` with the nearest Layers API release.

Please add vector type.